foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/OpenAiClient.java

```diff
@@ -105,6 +107,20 @@ public OpenAiChatCompletionOutput chatCompletion(
     return execute("/chat/completions", parameters, OpenAiChatCompletionOutput.class);
   }
 
+  /**
+   * Generate a completion for the given prompt.
+   *
+   * @param parameters the prompt, including messages and other parameters.
+   * @return the completion output
+   * @throws OpenAiClientException if the request fails
+   */
+  @Nonnull
+  public OpenAiChatCompletionStream<OpenAiStreamOutput> stream(
```

Comment:
(Comment/Major/Discussion)

This public method name is ambiguous:
* What does "stream" mean? Is it related to OpenAPI "stream" parameter or Java `Stream`?
* What entity is being "streamed"?

These question don't need to be answered here, but I think they should be apparent for the users of the API.

Reply:
Changed to `streamChatCompletion()`